beep/tests/test_features.py

```diff
@@ -309,7 +309,7 @@ def test_HPPCResistanceVoltageFeatures_class(self):
             self.assertEqual(featurizer.X.shape[1], 76)
             self.assertListEqual(
                 [featurizer.X.columns[0], featurizer.X.columns[-1]],
-                ["ohmic_r_d0", "D_8"],
+                ["r_c_0s_00", "D_8"],
```

Comment:
It would be helpful to add some testing values here to verify that the calculation is being done correctly.

Reply:
Added in some numerical values, and the calculation is the same with my notebook, but I encountered this error "ubuntu-latest_3_7 can't be loaded in"